code/game/machinery/telecomms/server_interface.dm

```diff
@@ -0,0 +1,86 @@
+/obj/machinery/computer/telecomms/server_interface
+	name = "telecommunications server interface"
+	icon_screen = "server_interface"
+	icon_keyboard = "teleport_key"
+	tcomms_linkable = 1 // That's the whole point of this.
+	light_color = LIGHT_COLOR_LIGHTBLUE
+	circuit = /obj/item/weapon/circuitboard/server_interface
+
+	var/list/ui_elements = list("text~Welcome. The telecomms server has not been configured to display a UI on this server interface. Press init to send \"init\" signal to server.","button~Init~init")
+
+/obj/machinery/computer/telecomms/server_interface/attack_hand(mob/user)
+	if(..())
+		return
+	interact(user)
+
+/obj/machinery/computer/telecomms/server_interface/attackby(obj/item/W, mob/user, params)
+	if(istype(W,/obj/item/weapon/pen))
+		user << "<span class='notice'>You insert \the [W]'s tip into \the [src]'s reset hole.</span>"
```

Comment:
`to_chat(user, "<span class='notice'>You insert \the [W]'s tip into \the [src]'s reset hole.</span>")`